ivy_tests/test_ivy/test_frontends/test_torch/test_dropout_functions.py

```diff
@@ -50,3 +50,55 @@ def test_torch_dropout(
     for u in ret:
         # cardinality test
         assert u.shape == x.shape
+
+
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.dropout2d",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=50,
+        allow_inf=False,
+        min_num_dims=2,
+        max_num_dims=4,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    prob=helpers.floats(min_value=0, max_value=0.9),
+    training=st.booleans(),
+    data_format=st.sampled_from(["NCHW", "NHWC"]),
+    test_gradients=st.just(False),
+    test_with_out=st.just(False),
+)
+def test_torch_dropout2d(
+    *,
+    dtype_and_x,
+    prob,
+    training,
+    data_format,
+    test_flags,
+    backend_fw,
+    on_device,
+    fn_name,
+    ground_truth_backend,
+):
+    dtype, x = dtype_and_x
+    ret, gt_ret = helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=dtype,
+        test_flags=test_flags,
+        test_values=False,
+        on_device=on_device,
+        fw=backend_fw,
+        fn_name=fn_name,
+        x=x[0],
+        prob=prob,
+        training=training,
+        data_format=data_format,
```

Comment:
The input parameters need to be renamed here as well i.e. `x` will be `input`, `prob` to `p` and `data_format` will be removed since it doesn't exist in the original function's signature

Reply:
absolutely